assets/src/edit-story/components/popup/index.js

```diff
@@ -65,29 +66,155 @@ const Container = styled.div.attrs(({ x, y }) => ({
   }
 `;
 
-function Popup({ anchor, children, width = DEFAULT_WIDTH, isOpen }) {
+function getXTransforms(placement) {
+  // left & right
+  if (placement.startsWith('left')) {
```

Comment:
The tests currently fail because `placement` is undefined.

https://github.com/google/web-stories-wp/pull/1060/checks?check_run_id=646520216#step:8:22